pkgs/development/libraries/liquid-dsp/default.nix

```
@@ -11,10 +16,10 @@ stdenv.mkDerivation rec {
     sha256 = "0mr86z37yycrqwbrmsiayi1vqrgpjq0pn1c3p1qrngipkw45jnn0";
   };
 
-  nativeBuildInputs = [ autoreconfHook ];
+  configureFlags = lib.optionals stdenv.isDarwin [ "LIBTOOL=${cctools}/bin/libtool" ];
+  nativeBuildInputs = [ autoreconfHook cctools ];
```

I think cctools is only necessary for darwin
```suggestion
  nativeBuildInputs = [ autoreconfHook ] ++ lib.optionals stdenv.isDarwin [ cctools ];
```

---

yeah, missed this. Fixed now